src/liballoc/lib.rs

```diff
@@ -65,7 +65,7 @@
 #![allow(explicit_outlives_requirements)]
 
 #![warn(deprecated_in_future)]
-#![warn(intra_doc_link_resolution_failure)]
+#![deny(intra_doc_link_resolution_failure)]
```

Comment:
We compile with `-D warnings`, which does this even make a difference?

Reply:
We don't yet pass `-D warnings` to rustdoc when building the docs unfortunately.

Reply:
Ah I see. When/if you change this next time, please add a comment explaining that. :)